array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,10 @@
+/* exported initial */
+function initial(array) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i !== array.length - 1) {
```

Comment:
You can remove this conditional by just making your loop stop one index sooner 🤓